mix.exs

```diff
@@ -8,7 +8,7 @@ defmodule Dataloader.Mixfile do
     [
       app: :dataloader,
       version: @version,
-      elixir: "~> 1.12",
+      elixir: "~> 1.11",
```

Comment:
Right now, the minimum supported Elixir version with security patches is 1.11, see https://hexdocs.pm/elixir/1.15.7/compatibility-and-deprecations.html